scenarios/kubernetes_e2e.py

```diff
@@ -500,6 +571,45 @@ def create_parser():
         action='append',
         default=[],
         help='Send unrecognized args directly to kubetest')
+
+    # aws
+    parser.add_argument(
+        '--aws', action='store_true', help='E2E job runs in aws')
+    parser.add_argument(
+        '--aws-profile',
+        default=(
+            os.environ.get('AWS_PROFILE') or
```

Comment:
The default was so that if you had the AWS CLI all properly configured, the scenario would pick it up correctly. (So yes, it's the local developer case.)

As for `AWS_PROFILE` vs `AWS_DEFAULT_PROFILE`, that's just a big wart involving the AWS command line and the SDK, which were mismatched for a while.